airflow/ui/openapi-gen/requests/schemas.gen.ts

```diff
@@ -4290,7 +4290,7 @@ export const $TimeDelta = {
   properties: {
     __type: {
       type: "string",
-      title: "  Type",
+      title: "Type",
```

Comment:
```suggestion
      title: Type,
```
doest it need to be ?